cpp/cmake/thirdparty/get_kvikio.cmake

```diff
@@ -0,0 +1,33 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+# This function finds KvikIO
+function(find_and_configure_kvikio VERSION)
+
+  rapids_cpm_find(
+    KvikIO ${VERSION}
+    GLOBAL_TARGETS kvikio::kvikio
+    CPM_ARGS
+    GIT_REPOSITORY https://github.com/madsbk/kvikio.git
+    # GIT_REPOSITORY https://github.com/rapidsai/kvikio.git TODO: use this before merge of PR
+    GIT_TAG used_by_cudf_for_testing
+    # GIT_TAG branch-${VERSION} TODO: use this before merge of PR
```

Comment:
There looks to be some leftover cruft here?

Reply:
Fixed